state-chain/pallets/cf-ingress-egress/src/lib.rs

```diff
@@ -1147,76 +1132,30 @@ pub mod pallet {
 		#[pallet::weight(T::WeightInfo::contract_ccm_swap_request())]
 		pub fn contract_ccm_swap_request(
```

Comment:
Yeah let's just merge these. They can only be called via witnessing so for backwards-compatibility / to avoid migration headaches, we could keep and deprecate the old ones and add a new one. Then we can remove the new one at a later release.

Reply:
> They can only be called via witnessing

Doesn't this mean we don't care about compatibility and can remove old extrinsics right away? New engines will only run after the runtime upgrade and will use the new api, right?

Reply:
Haven't we already broken compatibility by adding new parameters?

Reply:
As discussed - I guess we did 😅 

But given that we are now aware of this, let's avoid breaking - it's easy to do.